docs/source/guide/ddd-2-use-case.myst

```diff
@@ -12,3 +12,20 @@ kernelspec:
 ---
 
 # When should I use DDD?
+
+## Advantages
+
+Dynamical decoupling is a technique that has been originally devised to shield a quantum system from quantum noise coming from an environment, using open-loop control techniques :cite:`Viola_1999_PRL`. The crucial idea behind dynamical decoupling is to intervene with control pulses on timescales that are faster than those of the system-environment interaction.
+
+The theory of dynamical decoupling was originally devised at the pulse level and can include environments with memory effects. It can thus be effective also against structured baths or so called color-noise environments.
+
+Dynamical decoupling is also effective for one-shot quantum computing algorithms, i.e., it finds application beyond algorithms that just require expected values, like variational quantum algorithms.
+
+
+
+
+## Disadvantages
+
+Dynamical decoupling is generally applied at the pulse level. Mitiq provides it at the gate-level. In any case, native-gate control is required, to be able to know that the actual gates providing the decoupling sequence are run on the quantum processor or simulator.
```

Comment:
```suggestion
Dynamical decoupling is generally applied at the pulse level. Mitiq provides it at the gate-level. For this reason, it may be difficult to know and control what decoupling sequences are actually run on the quantum processor. A way to partially alleviate this issue, is to use DDD sequences built from the native gate set of the quantum backend.
```